### PR TITLE
Stop using broadcast recipient count 

### PR DESCRIPTION
**Pull request by norkans7**
Fixes https://github.com/rapidpro/rapidpro/issues/1096

### CODE REVIEW

File: temba/msgs/models.py [hunk 1]
```diff
@@ -280,6 +276,23 @@ def get_messages(self):
     def get_message_count(self):
         return BroadcastMsgCount.get_count(self)
 
+    def get_recipients_counts(self):
+        if self.status in (WIRED, SENT, DELIVERED):
+            return {"recipients": self.get_message_count(), "groups": 0, "contacts": 0, "urns": 0}
+
+        groups_count = self.groups.count()
+        contacts_count = self.contacts.count()
+        urns_count = self.urns.count()
```
Comment (nicpottier) [inline temba/msgs/models.py]: Small English thing, for something like a count use `group_count` not `groups_count`.

File: temba/msgs/models.py [hunk 2]
```diff
@@ -280,6 +276,23 @@ def get_messages(self):
     def get_message_count(self):
         return BroadcastMsgCount.get_count(self)
 
+    def get_recipients_counts(self):
+        if self.status in (WIRED, SENT, DELIVERED):
+            return {"recipients": self.get_message_count(), "groups": 0, "contacts": 0, "urns": 0}
+
+        groups_count = self.groups.count()
```
Comment (nicpottier) [inline temba/msgs/models.py]: Are these prefetched on this view?

**Comment by norkans7** [comment]
Still figuring out how to make the builder test the PO files properly, I disabled that temporary

**Comment by norkans7** [comment]
Still figuring out how to make the builder test the PO files properly, I disabled that temporary

**Comment by nicpottier** [comment]
Realized we actually need to do this in two commits / pushes. The migration to remove the field should come AFTER we switch to not using it.

File: temba/msgs/models.py [hunk 3]
```diff
@@ -280,6 +280,23 @@ def get_messages(self):
     def get_message_count(self):
         return BroadcastMsgCount.get_count(self)
 
+    def get_recipients_counts(self):
```
Comment (nicpottier) [inline temba/msgs/models.py]: change to `get_recipient_counts` and I'll merge this.

File: .travis.yml [hunk 4]
```diff
@@ -105,7 +105,8 @@ script:
 
   # fail if there are changes
   - git diff --numstat
-  - git diff --numstat | awk '$1 > 1 && $3 ~ /locale/ { print "makemessages needs to be rerun, see .travis.yml for command"; exit 1 }'
+  - git diff locale/
+  - git diff -U0 --summary --color=never | awk '/^+"/ && $0 !~ /POT-Creation-Date/ && $0 !~ /fuzzy/ { print $0, "\n\nMake messages needs to be rerun, see travis.yml for command"; exit 1; }'
```
Comment (rowanseymour) [inline .travis.yml]: just curious but what necessitated these changes?

Comment (norkans7) [reply .travis.yml]: because some PO files had more that 1 lines modified and the diff were just the `# fuzzy` lines

Comment (rowanseymour) [reply .travis.yml]: Think I have same problem now in https://github.com/nyaruka/rapidpro/pull/2615

Comment (nicpottier) [reply .travis.yml]: Wish I knew what the hell was causing that, but ya, this removes those spurious diffs.

**Pull request merged**